### PR TITLE
feat(modal): Allow not to use ajax

### PR DESCRIPTION
**Pull request by do3cc**
This patch offers two things.
1. Allow not to use ajax for forms. Useful for the delete confirmation
   popup
2. If a page does not have a base tag, find the content url from
   $(body).data('base-url')
   The data attribute has been added in plone.app.layout 2.5.1.dev0

Background on the delete confirmation:
We wanted to just redirect on success, but this looses features with ajax.
There is no way to get the statusmessage. The statusmessage is set on the
302 reply, then jquery directly follows the redirect and receives the
information that it shall delete the cookie. Now the success handler has
no trace of the status message. After thinking of patching statusmessage
to maybe add the status message just once as a response header field, I
realized I could just not use ajax on form submit since it offers no
advantage for this use case.

### CODE REVIEW

**Comment by jensens** [comment]
whats the state now here?

**Comment by do3cc** [comment]
This is not a PR anymore, it is just a token that people can use as  an example to claim that PRs don't work or to claim something about the state of mockup (That it needs more love(rs)). I don't plan to rebase this thing *again*.

On 26. April 2015 21:02:00 MESZ, "Jens W. Klein" <notifications@github.com> wrote:
>whats the state now here?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/plone/mockup/pull/370#issuecomment-96420967

-- 
Diese Nachricht wurde von meinem Mobiltelefon mit Kaiten Mail gesendet.

**Comment by do3cc** [comment]
It needs to be said more often, but thank you Jens for all your gardening effort on github

On 26. April 2015 21:25:49 MESZ, Patrick Gerken <gerken@patrick-gerken.de> wrote:
>This is not a PR anymore, it is just a token that people can use as  an
>example to claim that PRs don't work or to claim something about the
>state of mockup (That it needs more love(rs)). I don't plan to rebase
>this thing *again*.
>
>On 26. April 2015 21:02:00 MESZ, "Jens W. Klein"
><notifications@github.com> wrote:
>>whats the state now here?
>>
>>---
>>Reply to this email directly or view it on GitHub:
>>https://github.com/plone/mockup/pull/370#issuecomment-96420967
>
>-- 
>Diese Nachricht wurde von meinem Mobiltelefon mit Kaiten Mail gesendet.

-- 
Diese Nachricht wurde von meinem Mobiltelefon mit Kaiten Mail gesendet.

**Comment by jensens** [comment]
so i close this, its accessible then anyway

**Comment by jensens** [comment]
@kiorky why reopen? whats the plan with this one?

**Comment by do3cc** [comment]
I'll unsubscribe myself from notifications for this. @kiorky I assume that you want to take over, as you reopened this PR

**Comment by vangheem** [comment]
I think this https://github.com/plone/mockup/commit/43e1bb9a716e4d3ecf710d714816f2365412cb17 supersedes this right? Here is what the configuration looks like: https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/profiles/default/actions.xml#L192

I'm going to close. Please re-open if I misunderstand.
